test/overlay.html

```diff
@@ -152,151 +151,235 @@
             contextmenu();
           });
 
-          it('should be positioned on click target', done => {
-            listenOnce(overlay, 'vaadin-overlay-open', () => {
-              const rect = overlay.getBoundingClientRect();
-              expect(rect.left).to.eql(menu._phone ? 0 : 10);
-              expect(rect.top).to.eql(menu._phone ? 0 : 10);
-              done();
-            });
+          ['ltr', 'rtl'].forEach(direction => {
+
+            describe(`[dir=${direction}] opening`, () => {
+              let isRTL;
+              before((done) => {
+                // debugger;
+                isRTL = direction === 'rtl';
+                if (isRTL) {
+                  document.documentElement.setAttribute('dir', 'rtl');
```

Comment:
It can be `setAttribute('dir', direction)`, so `isRTL` can be removed.
Why do we need to set margin to zero in `rtl`?

Reply:
The margin was affecting the tests that uses `getBoundingClientRect`. While debugging, I could see that the overlay was positioned on the right place, but when I checked the bounding rect, it was always off by a bit and when margin was 0, that didn't happen anymore.